### ISSUE TITLE
WET 4 - audio multimedia player 

### ISSUE THREAD

**Issue by rcamelin** [wet-boew/wet-boew#7184]
This not an issue but a question...
I am using WET 4 (Intranet) as my framework for building courses. I am using the WET audio multimedia player and would like to have the player autoplay the audio on page load. I would like to do this via a script in order to give a 2 second delay after page load before the audio start to play. What command/function would I use so that it works in both html 5 and Flash fallback. I assume you have something built into the media player scripting to trigger on play button click I can reuse.

BTW it would be nice to have the CC feature added to the audio player.

**Comment by hsrudnicki** [comment]
@rcamelin : Are you still waiting for an answer from @LaurentGoderre @pjackson28 and/or others to this question?  Either way, please again update this issue's status whether this issue can be closed or otherwise? Thank you

**Comment by LaurentGoderre** [comment]
Hey, I haven't had the time to check but there is an autoplay attribute for HTML5 media element and I think the flash fallback should support it.

**Comment by rcamelin** [comment]
Hi, I ended up not using autoplay because I wanted the audio to start 2 seconds after the page loads. With autoplay it would start playing on page load. To get what I needed, I just scripted a delayed click action on the playpause button of the media player to start it.

 

But there is a feature I would like to see added to both the audio and video media player, the ability to play segments of the audio or video, by specifying a start and end time within the audio/video file. Something similar to the cue points  in the video player but with not only the start cue point but also a specified end point. In other word give the user the choice to  play the whole clip or play select chapters/segments. Sometimes it is not practical to break up a clip, or is not resource efficient to have two version of the same clip (whole and segmented).
